### ISSUE TITLE
Windows installer hangs [Kodi v20.1]

### ISSUE THREAD

**Issue by Hub-O-Gits** [xbmc/xbmc#23254]
Kodi = v20.1 x64
Env = Windows 7 x64

There appears to be an issue with the Kodi Windows installer (kodi-20.1-Nexus-x64.exe): During setup, if you de-select the MS Visual C++ packages or choose "Minimal" from the drop-down menu on the "Choose Components" screen, it apparently attempts to install those libraries anyway, and regardless, it hangs on the final setup screen, necessitating an End Task of the installer via Task Manager.

**Comment by CrystalP** [comment]
Going to remove the Ignored rules tag since the problem happens in the installer and the debug log is not relevant.

It's going to be difficult to troubleshoot, I don't think anyone in the team is still on Windows 7. This works fine in Windows 10.

And it's low priority since there is a workaround of letting the runtime install.

Is there a reason you didn't want the VC runtime installed?

**Comment by Hub-O-Gits** [comment]
I just deselected it because it was already present on the system. But perhaps the average user wouldn't bother checking. And of course there's no harm - if not better in some cases - to let it install anyway. The only real issue is that the installer hung at the end. But I tested on a Win 10 system today, deselected the VCR, and the installer didn't hang - which is most important since Win 10 is the de facto scenario at present.

I also re-tested the installer on a Win 7 system, but included the VCR this time, and the installer didn't hang there. So what remains then - installation on a Win 7 system where the VCR was deselected - is now a relatively fringe scenario.

I initially figured I should report it in case there was an issue with the installer routine, that could also affect Win 10 systems. I just didn't have that test bed available at the time.

So long story short then: never mind...

**Comment by CrystalP** [comment]
OK I'll close this then.
FYI version 20 Nexus (the current stable) is the last one that supports Windows 7.
v21 will be Windows 8.1 and above only.

**Comment by thexai** [comment]
Out of curiosity I tried to reproduce it yesterday on a Windows 7 machine and it worked correctly without installing VC runtimes, so it's not a general failure but probably only on that PC or due to a conflict with other software (antivirus, etc.)
